qualtran/_infra/controlled.py

```diff
@@ -60,17 +72,21 @@ class CtrlSpec:
             are active.
     """
 
-    qdtype: QDType = attrs.field(default=QBit())
-    cvs: NDArray[int] = attrs.field(
-        default=1,
-        converter=lambda cvs: np.array(cvs)
-        if isinstance(cvs, (int, np.integer))
-        else np.asarray(cvs),
+    qdtype: Tuple[QDType, ...] = attrs.field(
```

Comment:
change attribute name to `qdtypes`?

Reply:
anyways: update docstring

Reply:
renamed to `qdtypes` and updated docstring